manager/controllers/app/blueprint_controller.go

```diff
@@ -132,7 +132,7 @@ func (r *BlueprintReconciler) deleteExternalResources(blueprint *app.Blueprint)
 
 func (r *BlueprintReconciler) applyChartResource(log logr.Logger, chartSpec app.ChartSpec, args map[string]interface{}, blueprint *app.Blueprint, releaseName string) (ctrl.Result, error) {
 	log.Info(fmt.Sprintf("--- Chart Ref ---\n\n%v\n\n", chartSpec.Name))
-	kubeNamespace := blueprint.Namespace
+	kubeNamespace := utils.GetDataAccessModuleNamespace()
```

Comment:
Just a suggestion to avoid confusion, I think this method should be named GetDefaultDataAccessModuleNamespace(). The data access module namespace will either be the default configure when Fybrik is deployed OR what is specified with the label. For pass 1, when the label is specified these will be different. The helm release will be created in the default data access module but the artifacts for the data access module will be created in the namespace specified in the label.  When we get to pass 2, this will no longer be the case.

Reply:
Renamed to GetDefaultModulesNamespace()